### DISCUSSION TITLE
How Budibase count pricing per user if hosted on premise?

### DISCUSSION THREAD

**Discussion by NorraSains** [Budibase/budibase#11491] [General]
I would like to introduce Budibase to be adopted by my company. But there is a question from my management that I don't know the answer. The question was how this platform counts users if hosted on-premises. Hopefully, someone can advise.

**Comment by jomobudib** [comment]
Hey @NorraSains, to clarify the question - are you asking how Budibase knows how many users your self-hosted instance has, or are you asking how we charge for users?

If it's the former, your self-host instance will 'communicate' with our licensing server, letting it know how many registered users are on your Budibase instance. This is unless you're on Enterprise and wish to have an air-gapped instance set-up.

If it's the latter, on the self-host paid-tiers we charge per user the same as we do for cloud-based customers: see [here](www.budibase.com/pricing). In some circumstances there can be flexibility with the per-user model for Enterprise agreements.

If you or your management would like to chat through this in more detail (and with specifics to your situation), feel free to drop me an email on joe.moss@budibase.com.

Best,
Joe
